CHANGELOG.md

```diff
@@ -8,9 +8,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [next rel] - TBD
 ### Added
 - AdaScale: Added gradient accumulation feature
+- AdaScale: Added support of torch.lr_scheduler
```

Comment:
Shall we add a link to the PR for these additions?

Reply:
sure